src/components/ColorPicker/CustomColorPicker.tsx

```diff
@@ -55,10 +57,11 @@ export const CustomColorPicker: FC<Omit<ColorPickerProps, "palette">> = ({
             <div className="tw-flex tw-gap-2 tw-w-full tw-h-[200px]">
                 <div className="tw-relative tw-flex-grow tw-overflow-hidden tw-rounded">
                     <Saturation
-                        hsl={hsl}
-                        hsv={hsv}
+                        rgb={parsedColor.toRgb()}
+                        hsl={parsedColor.toHsl()}
+                        hsv={parsedColor.toHsv()}
```

Comment:
You could create variables `rgb, hsl, hsv` and reuse below

Reply:
Added